cmd/goal/messages.go

```diff
@@ -24,8 +24,8 @@ const (
 	errorGenesisIDFail          = "Error determining kmd folder (%s). Ensure the node is running in %s."
 	errorDirectoryNotExist      = "Specified directory '%s' does not exist."
 	errorParseAddr              = "Failed to parse addr: %v"
-	errorNonPrintableCharacters = "One or more non-printable characters were omitted from the following error message:"
-	infoNonPrintableCharacters  = "One or more non-printable characters were omitted from the subsequent line:"
+//	errorNonPrintableCharacters = "One or more non-printable characters were omitted from the following error message:"
```

Comment:
just remove them?

Reply:
Moved then to the report package

https://github.com/algorand/go-algorand/blob/ea410c8bd25e800498c20272ac2c0950d7adf675/cmd/util/report/messages.go#L17-L22

Reply:
> just remove them?

oh, it seems you meant "remove them instead of out-comment them".

k.